src/components/card/card.tsx

```diff
@@ -91,7 +91,7 @@ type EuiCardProps = Omit<CommonProps, 'aria-label'> & {
   /**
    * Requires a <EuiIcon> node
```

Comment:
Can we update this comment 
```suggestion
   * If provided, should be an `<EuiIcon>` node
```